common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -95,27 +143,15 @@ class ConditionalModule(ConditionalFields, XModule):
         'voted': 'voted'  # poll_question attr
     }
 
-    def _get_condition(self):
```

Comment:
Yes, I see  this is a real problem for described case. 
Would it be right decision to remove default value for the following fields 'conditional_attr', 'conditional_value' and implement this functionality the same way as in (https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/conditional_module.py#L202)